practice-three/src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="./style/style.css" />
+ </head>
+ <body>
+   <!-- START HEADER -->
+  <header class="st">
+    <div class="header-container">
+      <div class="crossbar">
+        <div class="navbar">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </div>
+        <nav class="site-nav">
+          <ul class="nav-menu">
+            <li class="menu-item">
+              <a href="javascript:void(0);" class="menu-item-link">Homepage</a>
+            </li>
+            <li class="menu-item">
+              <a href="javascript:void(0);" class="menu-item-link">Product</a>
+            </li>
+            <li class="menu-item">
+              <a href="javascript:void(0);" class="menu-item-link">Technologies</a>
+            </li>
+            <li class="menu-item">
+              <a href="javascript:void(0);" class="menu-item-link">Member</a>
+            </li>
+            <li class="menu-item">
+              <a href="javascript:void(0);" class="menu-item-link">About us</a>
+            </li>
+          </ul>
+          <button class="btn nav-btn"  type="button">Register</button>
```

Comment:
nav-btn is not suitable name, pls try another. Btw, space between class attribute and type attribute are 2 space, pls remove one